package.json

```diff
@@ -21,14 +21,16 @@
   "license": "MIT",
   "homepage": "",
   "dependencies": {
+    "@lifeomic/attempt": "^3.0.0",
     "bluebird": "^3.3.1",
-    "nxus-core": "^4.0.0",
+    "nxus-core": "^4.1.0",
     "nxus-router": "^4.0.0",
     "nxus-storage": "^4.0.0",
     "nxus-templater": "^4.0.0",
+    "nxus-tester-jest": "^4.0.9",
```

Comment:
Is there some reason nxus-tester-jest is in dependencies and not devDependencies?

Reply:
Nope, new commit cleans up dev deps.